debian/copyright

```diff
@@ -3,13 +3,14 @@ Upstream-Name: UCX
 Source: http://www.openucx.org
 
 Files: *
-Copyright (c) 2014-2015      UT-Battelle, LLC. All rights reserved.
-Copyright (c) 2014-2015      NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+Copyright:
+ 2014-2015      UT-Battelle, LLC. All rights reserved.
+ 2014-2015      NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
 License: BSD
  Redistribution and use in source and binary forms, with or without 
  modification, are permitted provided that the following conditions 
  are met:
- 
+ .
```

Comment:
why is this needed?

Reply:
This is part of the format. It is also used in the description in debian/control .
See https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#formatted-text
https://www.debian.org/doc/debian-policy/ch-controlfields#description